### DISCUSSION TITLE
Privacy Problem

### DISCUSSION THREAD

**Discussion by Havouza15** [immich-app/immich#7112] [General]
I understand the idea with Immich to have a self stored alternative to Google and such. We are a groups of friends and neighbors who want to use it like that and has installed it on a vps. So far it works without a problem but there is one thing that make it unusable in bigger groups. 
It is the partner sharing. Its a nice idea but it breaks all possible privacy because every user can see all other users email addresses. Its ofc ok if f.ex a family use it but there is a big interest to use it in our small village with about 300 households. 
Could it be possible to have a feature where the user can decide if he want to share his email or not.

**Comment by mrrfv** [comment]
You can work around this by giving each user a randomly generated or internal e-mail address (it doesn't have to be real). This shouldn't cause problems with Immich since it has no SMTP configuration to use for sending emails anyway.

**Comment by martabal** [comment]
Maybe we can introduce a new username field and make the full name optional

**Comment by DutchComputerKid** [comment]
I was about to add a similar point but yeah. I would like to share Immich with more photography peeps I know but the sharing feature either shows everyone or no one, with no in between. I would also like to see select-able sharing preferences or/and 'show my user to others' as it would be instead of clicking share and seeing everyone.
